src/contracts/Limitation/Target/DestinationLocation.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Core\Limitation\Target;
+
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\SPI\Limitation\Target;
+use eZ\Publish\SPI\Persistence\ValueObject;
+
+/**
+ * Location Limitation target.
+ */
+final class DestinationLocation extends ValueObject implements Target
+{
+    /** @var int */
+    private $id;
```

Comment:
Please consider renaming `$id` property to one of the following suggestions 

```suggestion
    private $locationId;
```

```suggestion
    private $targetLocationId;
```

Reply:
> Please consider renaming `$id` property to one of the following suggestions

:+1: :point_up: